examples/demo-apps/apple_ios/ExecuTorchDemo/ExecuTorchDemo.xcodeproj/project.pbxproj

```diff
@@ -806,7 +806,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/pytorch/executorch";
 			requirement = {
-				branch = latest;
+				branch = 0.4.0;
```

Comment:
Should be 0.4? https://github.com/pytorch/executorch/actions/runs/10925055073/job/30325617162?pr=5469#step:9:18833